README.md

```diff
@@ -1435,6 +1435,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [Vaibhav Wani](https://github.com/Vaibhavwani11)
   - [Vaibhavi Jha](https://github.com/vaibhavijha)
   - [Vaishnav Shyam](https://github.com/vaishnav-sh)
+  - [Vaishnav Nichal](https://github.com/vaishnavnichal)
```

Comment:
Could you please add your name in alphabetical order!

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Inefficient